korman/operators/op_mesh.py

```diff
@@ -118,27 +135,26 @@ def create_flare_objects(self):
         flare_plane.plasma_modifiers.fademod.bounds_center = True
 
     def find_create_material(self):
-        material_base_name = "AUTOGEN_FLARE"
-        # If one or more auto-generated flare materials already exist, use the newest one
-        matlist = [mat for mat in bpy.data.materials if mat.name.startswith(material_base_name)]
-        if matlist:
-            matlist.sort()
-            return matlist[-1]
-
-        # Generate a new flare material and texture
-        auto_mat = bpy.data.materials.new(name=material_base_name)
-        auto_mat.use_shadeless = True
-        auto_mat.use_shadows = False
-        auto_mat.use_cast_shadows = False
-
-        auto_tex = bpy.data.textures.new(name=material_base_name, type="IMAGE")
-        auto_tex.use_alpha = True
-        auto_tex.plasma_layer.skip_depth_write = True
-
-        new_slot = auto_mat.texture_slots.add()
-        new_slot.texture = auto_tex
-        return auto_mat
+        # If the selected flare material exists, use it
+        auto_mat = bpy.data.materials.get(self.flare_material_name)
```

Comment:
IIRC blender's `bpy_prop_collection` will raise an exception on error if a default is not provided.